apps/preview/nuxt/assets/css/tailwind.css

```diff
@@ -1,4 +0,0 @@
-@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Display:wght@300;400;500;700&family=Red+Hat+Text:ital,wght@0,400;0,500;0,700;1,300&display=swap');
```

Comment:
With this change there is no styling, question why its removed? wrong font 
![image](https://github.com/vuestorefront/storefront-ui/assets/2566152/ccca205f-c49e-49ae-8632-8d50acfeb1d8)

Reply:
fixed 1b24814a10c85c7ba4c482d507333c8bfc9dd6f9